alembic/alembic.ini

```diff
@@ -0,0 +1,74 @@
+# A generic, single database configuration.
+
+[alembic]
+# path to migration scripts
+script_location = .
+
+# template used to generate migration files
+# file_template = %%(rev)s_%%(slug)s
```

Comment:
Could we update this to format like ActiveRecord migrations?  I think it's useful to know when the migration was created, and it's useful to see the migrations chronologically.  

```
file_template = %%(year)d%%(month).2d%%(day).2d%%(hour).2d%%(minute).2d%%(second).2d_%%(slug)s
```